common/static/common/js/karma.common.conf.js

```diff
@@ -102,27 +124,60 @@ function junitSettings(config) {
     };
 }
 
-var getPreprocessorObject = function (files) {
-    var preprocessFiles = {};
+var normalizePathsForCoverage = function(files, normalize_func) {
+    var filesForCoverage = {};
 
     files.forEach(function (file) {
```

Comment:
I'm finding this logic hard to read. Can you name this anonymous function and give it a docstring.